Sources/ContentsquareInstance.swift

```diff
@@ -70,6 +103,19 @@ public class ContentsquareInstance: ContentsquareCommand {
     public func optOut() {
         Contentsquare.optOut()
     }
+    
+    private func parseIndex(_ value: Any?) -> UInt32? {
```

Comment:
Since we created this method, we could use it also for `sendDynamicVar` like so:

```swift
    public func sendDynamicVar(dynamicVar: [String: Any]) {
        dynamicVar.forEach { key, value in
            if let value = value as? String {
                let dynamicVar = DynamicVar(key: key, value: value)
                Contentsquare.send(dynamicVar: dynamicVar)
            } else if let value = parseIndex(value) {
                let dynamicVar = DynamicVar(key: key, value: value)
                Contentsquare.send(dynamicVar: dynamicVar)
            } else {
                print("Incorrect format of value: \(value). Value should be String or UInt32.")
            }
        }
    }
```